CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- `/whoami` API endpoint now produce audit events.
```

Comment:
@shulifink please review

Reply:
Shouldn't this change be placed in Appliance repo CHANGELOG.md? @netacoral